flowmachine/tests/test_majority_location.py

```diff
@@ -0,0 +1,51 @@
+from flowmachine.features import DayTrajectories, daily_location
+from flowmachine.features.location.majority_location import MajorityLocation
+from flowmachine.features.utilities.subscriber_locations import SubscriberLocations
+from flowmachine.features.subscriber.location_visits import LocationVisits
+from pandas import DataFrame as df
+from pandas.testing import assert_frame_equal
+
+import pytest
+
+
+@pytest.fixture
+def location_visits(flowmachine_connect):
+    lv = LocationVisits(
+        DayTrajectories(
+            daily_location("2016-01-01"),
+            daily_location("2016-01-02"),
+            daily_location("2016-01-03"),
+            daily_location("2016-01-04"),
+        )
+    )
+    yield lv
+
+
+def test_majority_location(get_dataframe, location_visits):
```

Comment:
Might be good to add a test that the `column_names` property matches the result columns